### DISCUSSION TITLE
No WIFI after flash

### DISCUSSION THREAD

**Discussion by walnut-engineering** [ludwig-v/wireless-carplay-dongle-reverse-engineering#169] [Q&A]
I have a carkitlink v4 (labelled A15 2.3 on PCB silkscreen) which I have flashed both MX25L12835F_CFW_20210830_V4.bin and MX25L12835F_CFW_20210830.bin, FW seems to boot fine - see the red/green flash for some seconds on first boot then solid red.  No WIFI or Bluetooth comes up however.  I can run scripts via USB and tried the Device_info scipt; adding 'echo | dmesg'.. wifi is broadcom with ID=0x4335; noticed script had entry for 4354 but not 4335, however looks to be the same driver.  Please see U2W.txt:

[U2W.txt](https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/files/11336058/U2W.txt)

**Comment by Qhilm** [comment]
Did you ever solve this?
